static/js/timerender.ts

```diff
@@ -68,8 +82,7 @@ export function render_now(time: Date, today = new Date()): TimeRender {
     // render formal time to be used for tippy tooltip
     // "\xa0" is U+00A0 NO-BREAK SPACE.
     // Can't use &nbsp; as that represents the literal string "&nbsp;".
-    const formal_time_str = format(time, "EEEE,\u00A0MMMM\u00A0d,\u00A0yyyy");
-
+    const formal_time_str = localizedFormat(time, {dateStyle: "full"});
```

Comment:
Would it be cleaner to just call `format(user_settings.default_language, {dateStyle: "full"})`? 

Mostly thinking about this in the context that we rename that function to be just about dates. (Could also be very reasonable to just extract a second parallel function that is just about "full date/time pairs" -- from a codebase readability standpoint, that might be considerably nicer than having individual code paths pass arcane `dateStyle` settings around)